.gitignore

```diff
@@ -1 +1,5 @@
-node_modules/
\ No newline at end of file
+node_modules/
+# Ignore compiled files
+src/**/*.js
```

Comment:
this breaks the plugin! ignoring the *.js files breaks the plugin as those are supposed to be executed by AutoRest core. The plugin is acquired by pointing `npm` to this repo, so all the bits must be there. Unless you create an npm package (sucks as it has to be republished to get any changes you make out there) or make AR core compile TS files before launching a plugin, this should remain unchanged.